src/InfoMCStructHelper.cc

```diff
@@ -209,12 +232,13 @@ namespace mu2e {
     siminfo.valid = true;
     if(sp.genParticle().isNonnull())siminfo.gen = sp.genParticle()->generatorId().id();
     siminfo.proc = sp.creationCode();
+    siminfo.stop = sp.stoppingCode();
```

Comment:
Hi Sophie - I don't see a ```stop``` leaf in the ```SimInfo``` struct in this PR. Did you meain to include an updated SimInfo.hh?